HelpSource/Classes/SinOsc.schelp

```diff
@@ -13,51 +13,404 @@ Technically, code::SinOsc:: uses the same implementation as  link::Classes/Osc::
 
 subsection:: Other sinewaves oscillators
 
-LIST::
-## link::Classes/FSinOsc:: – fast sinewave oscillator
-## link::Classes/SinOscFB:: – sinewave with phase feedback
-## link::Classes/PMOsc:: – phase modulation sine oscillator
-## link::Classes/Klang:: – bank of sinewave oscillators
-## link::Classes/DynKlang:: – modulable bank of sinewave oscillators
-::
+ LIST::
+ ## link::Classes/FSinOsc:: – fast sinewave oscillator
+ ## link::Classes/SinOscFB:: – sinewave with phase feedback
+ ## link::Classes/PMOsc:: – phase modulation sine oscillator
+ ## link::Classes/Klang:: – bank of sinewave oscillators
+ ## link::Classes/DynKlang:: – modulable bank of sinewave oscillators
+ ::
+
+This is the same as  link::Classes/Osc::  except that the table is
+a sine table of 8192 entries.
+
+From Osc Supercollider Documentation Description:
+
+Linear interpolating wavetable lookup oscillator with frequency and phase modulation inputs.
+
+This oscillator requires a buffer to be filled with a wavetable format signal. This preprocesses the Signal into a form which can be used efficiently by the Oscillator. The buffer size must be a power of 2.
+
+This can be acheived by creating a Buffer object and sending it one of the "b_gen" messages ( Buffer: -sine1, Buffer: -sine2, Buffer: -sine3 ) with the wavetable flag set to true.
+
+This can also be achieved by creating a Signal object and sending it the 'asWavetable' message, thereby creating a Wavetable object in the required format. Then, the wavetable data may be transmitted to the server using the Buffer: *sendCollection or Buffer: *loadCollection methods.
 
+////////////////////////////////
+////////////////////////////////
+////////////////////////////////
+
+About wavetable synthesis:
```

Comment:
This is tangential. SinOsc *internally* uses wavetable synthesis for efficiency, but I definitely wouldn't categorize SinOsc as a wavetable synthesis ugen.